scalardb-test/build.gradle

```diff
@@ -30,7 +31,18 @@ dependencies {
 
 shadowJar {
     mergeServiceFiles()
-    zip64 true
```

Comment:
I'm even sure if it is needed but is it deleted since you face some issues with it?
(I'm just wondering the reason for the deletion).

Reply:
I'm sorry, I can't reproduce this, but I removed this parameter because it didn't work when I ran it in a container once.